.gitlab-ci-check-c++.yml

```diff
@@ -0,0 +1,35 @@
+# .gitlab-ci-check-c++.yml
+#
+# This gitlab-ci template performs format checks using clang-format,
+#
+# Add it to the project in hand through Gitlab's include functionality
+#
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-c++.yml'
+#
+# The standard .clang-format from this repository is used unless the
+# project including this template has its own version in the root dir.
+#
+
+stages:
+  - test
+
+test:format:
+  stage: test
+  needs: []
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
```

Comment:
This was grabbed from the Golang format template. I think this is mainly to avoid failures on the saas branches, where we typically only merge, and it's too late to fix the problems by that point.

But you're right it probably isn't necessary for C++. Minimalism is better in this case.